index.js

```diff
@@ -371,7 +371,7 @@ class SortableFlatList extends Component {
         }}
         ref={this.measureContainer}
         {...this._panResponder.panHandlers}
-        style={styles.wrapper} // Setting { opacity: 1 } fixes Android measurement bug: https://github.com/facebook/react-native/issues/18034#issuecomment-368417691
+        style={[styles.wrapper, contentContainerStyle]} // Setting { opacity: 1 } fixes Android measurement bug: https://github.com/facebook/react-native/issues/18034#issuecomment-368417691
```

Comment:
this prop name collides with the underlying `FlatList` `contentContainerStyle`. I think that would be misleading/confusing for users.